doc/manual/packages.rst

```diff
@@ -570,11 +570,21 @@ going to have to do a *force push*:
   indicates that this should replace the ``fixbar`` branch found at
   ``myfork``.
 
-
 Creating a new Package
 ----------------------
 
-Guidelines for Naming a Package
+REQUIRE speaks for itself
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You should have a ``REQUIRE`` file in your package repo, with a bare minimum directive of what Julia version to include.
+As Julia moves to different versions, many packages include incorrect version requirements in their ``REQUIRE`` file. The most
+common mistake is to include pre-release versions of Julia after a release has been officially completed. If Julia has
+officially released version ``0.x``, for example, then all the Julia-specific requirements in the ``REQUIRE`` file should not
```

Comment:
Maybe reword this more along the lines of, `if Julia version 0.x has been released then you should use 0.x, not 0.x-. 0.x- indicates that you package is compatible with the yet....` and then the rest. Just so its more what you should do, rather than not do, is my thinking here.